mvnw

```diff
@@ -200,65 +200,6 @@ if [ -z "$BASE_DIR" ]; then
   exit 1;
 fi
 
-##########################################################################################
```

Comment:
Why are we changing the Maven wrapper?

Reply:
To provide latest solutions to users. Not also maven but also maven-plugins and SpotBugs are kept updated in this project.